build.sbt

```diff
@@ -12,7 +12,7 @@ scalaVersion in ThisBuild := "2.12.8"
 lagomServiceEnableSsl in ThisBuild := true
 val `hello-impl-HTTPS-port` = 11000
 
-resolvers in ThisBuild += Resolver.bintrayRepo("akka", "maven") // for snapshot akka-grpc
+val lagomGrpcTestkit = "com.lightbend.play" %% "lagom-javadsl-grpc-testkit" % "0.5.0+36-729a28ba+20190307-1146"
```

Comment:
It's ok, that Lagom module in group `com.lightbend.play`? I am surprised 😄

Reply:
Yeah, we're bringing the products closer... apparently have started sharing coordinates! 😄